src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -23,6 +24,43 @@ define([
         "text-warning": isDeploying
       });
 
+      var hasHealth = model.get("healthChecks") != null &&
+        model.get("healthChecks").length > 0;
+
+      var healthPieColors = [
+        "#7aff68", // healthy
```

Comment:
Please no colors in the js code and let's stick to the existing health color styles: https://github.com/mesosphere/marathon/blob/master/src/main/resources/assets/css/marathon.css#L551

Here maybe just add the following (please, remove the comments, those are for you):
```
.progress-bar-info {
  background-color: #21c0fb; // (blue) unscheduled - I'm not sure what to do here?
}

.progress-bar-error {
  background-color: #ff7d64; // (red) unhealthy - needs attention
}

.progress-bar-warning {
  background-color: #faef66; // (yellow) staged + over-capacity - needs a little attention
}

.progress-bar-success {
  background-color: #7aff68; // (green) healthy - will create attention, so I'm not sure that this is the best idea?
}

.progress-bar-unknown {
  color: #8e929b; // (grey) running - doesn't need attention
}
```